File: samples/csharp/src/Hashes.cs

using System;
using System.Collections.Generic;

// UI settings
// Expression: h1
//   Type: System.Collections.Generic.Dictionary<string, int>
//     Layout: Map
//     Save: Object Attributes
// Expression: hs1
//   Type: System.Collections.Generic.HashSet<string>
//     Layout: Set
//     Save: Object Attributes
class Hashes
{
    public static void RunMain(string[] args)
    {
        new Hashes().Start();
    }

    public void Start()
    {
        Dictionary<string, int> h1 = new Dictionary<string, int>();
        h1["1"] = 1;
        h1["2"] = 2;

        HashSet<string> hs1 = new HashSet<string>();
        hs1.Add("1");
        hs1.Add("2");

        Console.WriteLine("done");
    }
}

File: samples/ts/src/Hashes.ts
// UI settings
// Expression: h1
//   Type: Map
//     Layout: Map
//     Save: Object Attributes
// Expression: hs1
//   Type: Set
//     Layout: Set
//     Save: Object Attributes
// 
class Hashes {
    public static runMain() {
        new Hashes().start();
    }

    start() {
        // expression: h1
        const h1: Map<string, number> = new Map<string, number>();
        h1.set("1", 1);
        h1.set("2", 2);

        // expression: hs1
        const hs1: Set<string> = new Set<string>();
        hs1.add("1");
        hs1.add("2");

        console.log("done");
    }
}

Hashes.runMain();